.prettierignore

```diff
@@ -1,6 +1,7 @@
 # Needed for lint staged.
 .prettierignore
 yarn.lock
+.eartlyignore
```

Comment:
This was needed for lint staged to pass as prettier could not infer a parser for it and since lint staged explicitly pass files to prettier it failed and I could not commit without it.